force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1361,6 +1361,38 @@
         <shortDescription>EPAddDependentTask</shortDescription>
         <value>Add Dependent Task</value>
     </labels>
+    <labels>
+        <fullName>donorsGivenSummary</fullName>
+        <categories>Donors</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Component title</shortDescription>
+        <value>Given Summary</value>
```

Comment:
I think this should be "Giving" Summary, though we'll want to confirm with UX.

Reply:
@lparrott  You are correct. This was an error in the original quip design doc. All references to "Given Summary" should actually be "Giving Summary". Something to look for in api names, labels, class names, etc. Even the branch name is incorrect, though that's not worth changing at this point.

Reply:
Done